worker_tasks/categorize.py

```diff
@@ -279,15 +283,24 @@ def _assign_closest_articles(self, ngrams):
             max_relatedness_score = MINIMUM_RELATEDNESS_SCORE
             best_article = None
 
-            for article in self._memoized_related_articles[ambiguous_ngram]:
+            max_relatedness_score_without_parentheses = 0
+            best_article_without_parentheses = None
 
+            for article in self._memoized_related_articles[ambiguous_ngram]:
                 article_relatedness_score = self._get_relatedness_score(self._memoized_article_links[article], context_links)
 
                 if article_relatedness_score > max_relatedness_score:
                     max_relatedness_score = article_relatedness_score
                     best_article = article
 
-            if best_article:
+                if "(" not in article:
+                    if article_relatedness_score > max_relatedness_score_without_parentheses:
+                            max_relatedness_score_without_parentheses = article_relatedness_score
+                            best_article_without_parentheses = article
+
+            if best_article_without_parentheses:
+                assigned_articles.append(best_article_without_parentheses)
+            elif best_article:
                 assigned_articles.append(best_article)
 
         return assigned_articles
```

Comment:
Add a test that preload ```categorizer._memoized_related_articles``` that yields a best article where ```best article``` is better than ```best_article_without_parentheses```. Ex: a review of Home (2015 film).